modules/502-delivery/docs/USAGE_RU.md

```diff
@@ -0,0 +1,317 @@
+---
+title: "Модуль delivery: пример конфигурации"
+---
+
+- [Дисклеймер](#дисклеймер)
+- [Концепция](#концепция)
+- [Конфигурация](#конфигурация)
+  - [WerfSource CRD](#werfsource-crd)
+  - [Частичное использование WerfSource CRD](#частичное-использование-werfsource-crd)
+    - [Без репозитория Argo CD](#без-репозитория-argo-cd)
+    - [Без регистри для Image Updater](#без-регистри-для-image-updater)
+- [Публикация артефакта в registry](#публикация-артефакта-в-registry)
+- [Деплой](#деплой)
+- [Автообновление бандла](#автообновление-бандла)
+  - [Правила обновлений образов](#правила-обновлений-образов)
+  - [Настройки доступа к registry](#настройки-доступа-к-registry)
+    - [Индивидуальная настройка для Application](#индивидуальная-настройка-для-application)
+- [Особенности аутентификации CLI](#особенности-аутентификации-cli)
+- [Как создать OCI-репозиторий самостоятельно](#как-создать-oci-репозиторий-самостоятельно)
+  - [Веб-интерфейс и kubectl](#веб-интерфейс-и-kubectl)
+  - [Командная утилита `argocd`](#командная-утилита-argocd)
+
+## Дисклеймер
```

Comment:
Doesn't seem like the right word for documentation.

Reply:
I'm open to suggestions.